### PR TITLE
#RI-3633 - The largest value in graph is more than largest y-axis point

### PR DESCRIPTION
**Pull request by zalenskiSofteq**
#RI-3633 - The largest value in graph is more than largest y-axis point

### CODE REVIEW

File: redisinsight/ui/src/components/charts/area-chart/AreaChart.tsx [hunk 1]
```diff
@@ -56,6 +66,15 @@ const AreaChart = (props: IProps) => {
 
   const svgRef = useRef<SVGSVGElement>(null)
 
+  const getRoundedYMaxValue = (number: number): number => {
+    const numLen = number.toString().length
+    let dividerValue = '1'
+    for (let i = 0; i < numLen - 1; i++) {
+      dividerValue += 0
+    }
```
Comment (romansergeenkosofteq) [inline redisinsight/ui/src/components/charts/area-chart/AreaChart.tsx:74]: maybe just use
`const dividerValue = toNumber('1' + '0'.repeat(numLen - 1))`

File: redisinsight/ui/src/utils/formatBytes.ts [hunk 2]
```diff
@@ -23,3 +23,12 @@ export const formatBytes = (
     return '-'
   }
 }
+
+export const toBytes = (size: number, type: string): number | string => {
+  const key = SIZES.indexOf(type.toUpperCase())
+
+  if (typeof key !== 'boolean') {
```
Comment (romansergeenkosofteq) [inline redisinsight/ui/src/utils/formatBytes.ts]: when key can be a boolean?

File: redisinsight/ui/src/utils/tests/formatTypes.spec.ts [hunk 3]
```diff
@@ -48,3 +48,34 @@ describe('formatBytes', () => {
...
+  [1125899906842624, undefined, '1 PB'],
+  [1125899906842624, undefined, '1 PB'],
+  [1413004383087493, undefined, '1.255 PB'],
+  [1152921504606847000, undefined, '1 EB'],
+  [1152921504606847000, undefined, '1 EB'],
+  [1446916488281592800, undefined, '1.255 EB'],
+  [1.1805916207174113e+21, undefined, '1 ZB'],
+  [1.2089258196146292e+24, undefined, '1 YB'],
+  [1.2379400392853803e+27, undefined, '1024 YB'],
+]
```
Comment (romansergeenkosofteq) [inline redisinsight/ui/src/utils/tests/formatTypes.spec.ts]: I think no need here second item of array

File: redisinsight/ui/src/components/charts/area-chart/AreaChart.tsx (hunk 1 above)
Comment (zalenskiSofteq) [reply redisinsight/ui/src/components/charts/area-chart/AreaChart.tsx]: changed

File: redisinsight/ui/src/utils/formatBytes.ts (hunk 2 above)
Comment (zalenskiSofteq) [reply redisinsight/ui/src/utils/formatBytes.ts]: removed

**Review by romansergeenkosofteq — APPROVED** [review]

**Pull request merged**